buildSrc/src/test/java/org/elasticsearch/gradle/FileContentsTaskTests.java

```diff
@@ -16,10 +17,13 @@
 
     @Rule
     public TemporaryFolder temporaryFolder = new TemporaryFolder();
```

Comment:
Is it still used ?
@elasticmachine test this please

Reply:
yep to create the project in `createProject()`